notebooks/04-onnx-export.ipynb

```diff
@@ -284,141 +626,199 @@
+      "text/plain": [
+       "<Figure size 1600x1200 with 1 Axes>"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "%matplotlib inline\n",
     "\n",
+    "import matplotlib\n",
+    "import matplotlib.pyplot as plt\n",
+    "import numpy as np\n",
+    "import os\n",
     "\n",
-    "# All the providers we'll be using in the test\n",
-    "results = {}\n",
-    "providers = [\n",
-    "  \"CUDAExecutionProvider\",\n",
-    "  \"CPUExecutionProvider\",            \n",
-    "  \"TensorrtExecutionProvider\",\n",
-    "  \"DnnlExecutionProvider\",          \n",
-    "]\n",
     "\n",
-    "# Iterate over all the providers\n",
-    "for provider in providers:\n",
+    "# Compute average inference time + std\n",
+    "time_results = {k: np.mean(v.model_inference_time) * 1e3 for k, v in results.items()}\n",
+    "time_results_std = np.std([v.model_inference_time for v in results.values()]) * 1000\n",
     "\n",
-    "  # Create the model with the specified provider\n",
-    "  model = create_model_for_provider(\"onnx/bert-base-cased.onnx\", provider)\n",
+    "plt.rcdefaults()\n",
+    "fig, ax = plt.subplots(figsize=(16, 12))\n",
+    "ax.set_ylabel(\"Avg Inference time (ms)\")\n",
+    "ax.set_title(\"Average inference time (ms) for each provider\")\n",
+    "ax.bar(time_results.keys(), time_results.values(), yerr=time_results_std)\n",
+    "plt.show()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Quantization support from transformers\n",
     "\n",
-    "  # Keep track of the inference time\n",
-    "  time_buffer = []\n",
+    "Quantization enables the use of integers (_instead of floatting point_) arithmetic to run neural networks models faster. From a high-level point of view, quantization works as mapping the float32 ranges of values as int8 with the less loss in the performances of the model.\n",
     "\n",
-    "  # Warm up the model\n",
-    "  for _ in trange(10, desc=\"Warming up\"):\n",
-    "    model.run(None, inputs_onnx)\n",
+    "Hugging Face provides a conversion tool as part of the transformers repository to easily export quantized models to ONNX Runtime. For more information, please refer to the following: \n",
     "\n",
-    "  # Compute \n",
-    "  for _ in trange(100, desc=f\"Tracking inference time on {provider}\"):\n",
-    "    with track_infer_time(time_buffer):\n",
-    "      model.run(None, inputs_onnx)\n",
+    "- [Hugging Face Documentation on ONNX Runtime quantization supports](https://huggingface.co/transformers/master/serialization.html#quantization)\n",
+    "- [Intel Nervana details](https://nervanasystems.github.io/distiller/quantization.html)"
```

Comment:
I don't believe Nervana components are used in this notebook. To avoid confusion, I suggest changing this to "[Explanation of Quantization]"